storage/clickhouse/clickhouse.go

```diff
@@ -227,36 +229,38 @@ func (ch *clickHouse) scanSamples(rows *sql.Rows) ([]*prompb.TimeSeries, error)
 		res = append(res, ts)
 	}
 
+	ch.l.Infof("Used %d time series from %d time series returned by query", usedFingerprintCount, totalFingerprintCount)
```

Comment:
@srikanthccv when would they be different?

Reply:
Because some of them might not be not be in the time range selected for query. We discussed this and decided just to log the number of actually used time series from the response.

Reply:
ah..yes..I remember